src/bitbucket-pr.coffee

```diff
@@ -15,243 +11,267 @@
 # Author:
 #   tshedor
 
-url = require('url')
-querystring = require('querystring')
+DEFAULT_ROOM = process.env.HUBOT_BITBUCKET_PULLREQUEST_ROOM
+
+# Set default actions to announce
+DEFAULT_ANNOUNCE_OPTIONS = [
+  'created', 'updated', 'declined', 'merged',
+  'comment_created', 'approve', 'unapprove'
+]
+
+getEnvAnnounceOptions = ->
+  # Replace announce options if set in the environment
+  if process.env.HUBOT_BITBUCKET_PULLREQUEST_ANNOUNCE
+    process.env.HUBOT_BITBUCKET_PULLREQUEST_ANNOUNCE.replace(/[^a-z\,]+/, '').split(',')
+
+ANNOUNCE_OPTIONS = getEnvAnnounceOptions() ? DEFAULT_ANNOUNCE_OPTIONS
```

Comment:
I think this could be consolidated into one func without `DEFAULT_ANNOUNCE_OPTIONS`, i.e.

```
getEnvAnnounceOptions = ->
  # Replace announce options if set in the environment
  if process.env.HUBOT_BITBUCKET_PULLREQUEST_ANNOUNCE
    process.env.HUBOT_BITBUCKET_PULLREQUEST_ANNOUNCE.replace(/[^a-z\,]+/, '').split(',')
  # Fall back to default actions to announce
  else
    [ 'created', 'updated', 'declined', 'merged', 'comment_created', 'approve', 'unapprove' ]

ANNOUNCE_OPTIONS = getEnvAnnounceOptions()
```